### PR TITLE
Add an endpoint to retrieve all of an user's task scores.

### PR DESCRIPTION
**Pull request by veluca93**

### CODE REVIEW

**Review by dp1 — APPROVED** [review]

File: backend/terry/handlers/info_handler.py [hunk 1]
```diff
@@ -131,6 +131,30 @@ def get_user(self, user):
 
         return BaseHandler.format_dates(user, fields=["end_time"])
 
+    @Validators.contest_started
```
Comment (edomora97) [inline backend/terry/handlers/info_handler.py:134]:
```suggestion
    @Validators.contest_started
    @Validators.register_user_ip
```

Haven't tested it yet.

File: backend/terry/server.py [hunk 2]
```diff
@@ -48,7 +48,8 @@ def __init__(self):
         self.router = Map(
             [
                 Rule("/contest", methods=["GET"], endpoint="info#get_contest"),
-                Rule("/input/<input_id>", methods=["GET"], endpoint="info#get_input"),
+                Rule("/input/<input_id>",
```
Comment (edomora97) [inline backend/terry/server.py:51]: I don't know which formatter you used, but it doesn't look like the correct one. Can you remove these changes?

File: backend/terry/server.py [hunk 3]
```diff
@@ -60,7 +61,10 @@ def __init__(self):
                     methods=["GET"],
                     endpoint="info#get_submission",
                 ),
-                Rule("/user/<token>", methods=["GET"], endpoint="info#get_user"),
+                Rule("/scores/<token>",
```
Comment (edomora97) [inline backend/terry/server.py:64]: What about `/user/<token>/scores`?

**Review by edomora97 — CHANGES_REQUESTED** [review]

File: backend/terry/handlers/info_handler.py (hunk 1 above)
Comment (veluca93) [reply backend/terry/handlers/info_handler.py:134]: It seems to work.

**Review by edomora97 — APPROVED** [review]

**Pull request merged**